openapi/kas-fleet-manager-private-admin.yaml

```diff
@@ -199,6 +199,54 @@ paths:
             application/json:
               schema:
                 $ref: 'kas-fleet-manager.yaml#/components/schemas/Error'
+  '/api/kafkas_mgmt/v1/admin/kafkas/{id}/revoke_certificate':
+    post:
+      description: Revokes the certificate for the Kafka instance by id
```

Comment:
When we talk about "certificate" what are we talking about?

Reply:
Kafka tls certificate of course ;-)

Reply:
What is a kafka tls certificate?